src/integration/vliw-integration.ts

```diff
@@ -181,6 +181,7 @@ export class VLIWIntegration extends MachineIntegration {
             this.executionLoop(speed);
         } else {
             // tslint:disable-next-line:no-empty
+            //TODO: Should we show VLIWErrors and stop execution?
```

Comment:
We should! One of the _first-good-issue_ kind of features I have been thinking of is for exceptions to be printed to the user as an alert.

For example, Material UI users would use this kind of component, probably in the lower right corner, to notice the user of an error: https://mui.com/material-ui/react-alert/

Reply:
This TODO was referencing another issue, in this case, when we are in a normal execution, VLIWErrors stops (in reality pause) the execution and shows an alert. But, this TODO references the Batch execution part, this type of execution can not be paused in any way (also, it will be a pain point for the user stopping this execution). So, it will be interesting showing in the Batch results modal a message alerting the user that there was something that requires them attention (hazzards, breakpoints, etc).

Sorry, for the cryptic TODO.